modules/apps/layout/layout-page-template-admin-web-test/src/testIntegration/resources/com/liferay/layout/page/template/admin/web/internal/portlet/action/test/dependencies/import_export/page_templates/fragment/text_field/mapped_value/class_pk_reference/expected/fragment_available/page-templates/test-page-template-collection/test-page-template/page-definition.json

```diff
@@ -23,7 +23,34 @@
 								}
 							}
 						}
-					]
+					],
+					"styles": {
+						"backgroundImage": {
+						},
+						"borderRadius": "rounded-0",
```

Comment:
IMHO we should include all these attributes in the model.

Modelling `styles` as  a generic JSON object does not allow users to know which attributes are supported and allows performing validation.